web/ui/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ui",
+  "version": "0.0.0",
```

Comment:
IMO, we should keep the whole project with same version, so we should fill this with `0.5.0-SNAPSHOT`. WDYT

Reply:
then release guide about how to bump version part should be updated too cc @FMX

Reply:
I don't think so, the current version of the ui should not be consistent with the overall project, certainly does not mean that it will not be synchronized in the future

Reply:
why use a different version? does it mean that one webui can be used with different backend versions? where is the compatible matrix?